src/testdir/test_spellrare.vim

```diff
@@ -0,0 +1,65 @@
+" Test spell checking
+
+source check.vim
+CheckFeature spell
+
+func TearDown()
+  set nospell
+  " set 'encoding' to clear the word list
+  set encoding=utf-8
+endfunc
+
+" Test spellbadword() with argument, specifically to move to "rare" words
+" in normal mode.
+func Test_spellrareword()
+  set spell
+
+  " Create a small word list to test that spellbadword('...')
+  " can return ['...', 'rare'].
+  e Xwords
+  insert
+foo
+foobar/?
+foobara/?
+.
+   w!
```

Comment:
Instead of this, can you use the following?
```
   let lines =<< trim END
     foo
     foobar/?
     foobara/?
   END
   call writefile(lines, 'Xwords', 'D')
```